### ISSUE TITLE
Setting `verbose: true` causes an exception

### ISSUE THREAD

**Issue by garth** [krisk/Fuse#76]
```
Uncaught (in promise) TypeError: Illegal invocation(…)
    log @ fuse.js:24
    Fuse.search @ fuse.js:103
```

**Comment by kylehalleman** [comment]
I fixed this issue in PR #74 but forgot to update the version number in package.json, so npm doesn't show any update. The code is fixed. Will submit a PR to update the version number when I have access to my machine (or someone else can do it).

**Comment by kylehalleman** [comment]
#79 will reflect the fix
